crates/curp/src/members.rs

```diff
@@ -138,26 +144,31 @@ impl ClusterInfo {
     pub fn from_cluster(
         cluster: FetchClusterResponse,
         self_peer_urls: &[String],
+        self_client_urls: &[String],
         self_name: &str,
     ) -> Self {
         let mut member_id = 0;
-        let members = cluster
+        let sorted_self_addr = self_peer_urls.iter().cloned().sorted().collect::<Vec<_>>();
+        let members: DashMap<u64, Member> = cluster
             .members
             .into_iter()
             .map(|mut member| {
-                if member.peer_urls() == self_peer_urls {
+                if sorted_self_addr == member.peer_urls() {
```

Comment:
If self_addr is sorted, does the `peer_urls`'s ordering matter while being compared?

Reply:
Done